include/kodlab_mjbots_sdk/joint_moteus.h

```diff
@@ -71,12 +79,20 @@ class JointMoteus: public JointBase
             float pos_max = std::numeric_limits<float>::infinity(),
             float soft_start_duration_ms = 1,
             float moteus_kp = 0,
-            float moteus_kd = 0)
+            float moteus_kd = 0,
+            float temperature = 0,
+            float q_current = 0,
```

Comment:
Why is a state being set by the constructor?

Reply:
I misunderstood what we were talking about when we were talking earlier and now understand why you were so flabbergasted and agree.